ur10_moveit_config/launch/planning_context.launch

```diff
@@ -1,20 +1,16 @@
 <launch>
   <!-- By default we do not overwrite the URDF. Change the following to true to change the default behavior -->
   <arg name="load_robot_description" default="false"/>
-  <arg name="limited" default="false"/>
 
   <!-- The name of the parameter under which the URDF is loaded -->
   <arg name="robot_description" default="robot_description"/>
 
   <!-- Load universal robot description format (URDF) -->
-  <group if="$(arg load_robot_description)">
-    <param unless="$(arg limited)" name="$(arg robot_description)" command="$(find xacro)/xacro --inorder '$(find ur_description)/urdf/ur10_robot.urdf.xacro'" />
-    <param if="$(arg limited)" name="$(arg robot_description)" command="$(find xacro)/xacro --inorder '$(find ur_description)/urdf/ur10_joint_limited_robot.urdf.xacro'" />
-  </group>
+  <param if="$(arg load_robot_description)" name="$(arg robot_description)" command="xacro  '$(find ur_description)/urdf/ur10.xacro'"/>
```

Comment:
Why does this not just include the corresponding `load_*.launch`file?

Reply:
Because that would take this file further away from what is generated by the MSA, which would increase maintenance effort.

See https://github.com/ros-industrial/fanuc/pull/170 for a PR which first tried to do what you suggest here.